src/test/java/reposense/git/GitLsTreeTest.java

```diff
@@ -81,11 +79,8 @@ public void unix_cloneInvalidWindowsFilePaths_success() {
     /**
      * Clones a bare repo in {@code config} and verifies that the repo contains only file paths that are
      * compatible in Windows.
-     * @throws IOException if it fails to create/delete the folders.
-     * @throws InvalidFilePathException if the repository contains invalid file paths that are not compatible with
-     * Windows.
      */
-    private void validateFilePaths(RepoConfiguration config) throws IOException, InvalidFilePathException {
+    private void validateFilePaths(RepoConfiguration config) throws Exception {
```

Comment:
LGTM! Just one nit. Isn't the javadoc still suppose to mention `@throws Exception`? Not sure about this. Can ask someone else for this and proceed accordingly.

Reply:
Actually, in this case, `validateFilePaths` is a utility function that is used by test code (it's not a test itself).
Would it be better to keep the existing `throws` clause for such utility functions?
Keeping the existing `throws` clause may provide more clarity to developers who are writing tests using these functions.

We can use the generic `throws` clause for the tests themselves.
@fzdy1914 may we get your thoughts on this?

Reply:
I think that we should stick to the generic `Exception` class because the test method that calls it also throws only the generic `Exception`. 
There is little use of mentioning the specific exceptions thrown by the utility method if the test method itself doesn't mention them, is there?

Reply:
In my view, it may be useful to specify the exceptions thrown by the utility method. 
Reason: sometimes, test cases check whether certain exceptions are thrown during testing. When a developer is writing such tests cases, it could be useful to know what exceptions are thrown by the utility method.

Reply:
I agrees with what @anubh-v says.

Reply:
@yash-chowdhary has resolved these changes. 
May we get your review on the PR @fzdy1914 ? Thanks for your time